tough/src/schema/error.rs

```diff
@@ -93,6 +97,23 @@ pub enum Error {
     /// Unable to create a TUF target from anything but a file
     #[snafu(display("TUF targets must be files, given: '{}'", path.display()))]
     TargetNotAFile { path: PathBuf, backtrace: Backtrace },
+
+    /// Target doesn't have proper permissions from parent delegations
+    #[snafu(display("Invalid file permissions: {}", child))]
+    UnmatchedPath { child: String },
+
+    /// No valid targets claims target_url
+    #[snafu(display("Target file not delegated: {}", target_url))]
+    TargetNotFound { target_url: String },
```

Comment:
At first read, the docstring, display string, and variant name all disagree with each other